app/views/partials/_userCard.html.erb

```diff
@@ -0,0 +1,13 @@
+<% (1..items).each do %>
+  <div class="horizontal-panel flex">
+    <div class="card-photo border-black">
+      <!-- @TODO: add image here later --> 
+    </div>
+    <div class="card-contents flex items-center space-between w-full border-black"> 
+      <h1 class="card-title w-auto">Username</h1>
+      <div class="card-paragraph-container h-full flex flex-end w-auto">
+        <p class="card-paragraph">Number of posts: x</p>
+      </div>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] Folliowing best practices it's recommended  that displaying _users_ in the _index_ of views should look like:
```ruby
 <% @users.each do |user| %>
     # HTML Code Here displaying user.photo, user.name and user.posts_counter
    <% end %>
```

Reply:
This has been fixed. Thanks for picking it up.